### PR TITLE
version 0.5.1: GHC 8.6 support, stack support

### PR DESCRIPTION
**Pull request by ozzzzz**

### CODE REVIEW

File: src/Database/PostgreSQL/ORM/Validations.hs [hunk 1]
```diff
@@ -9,16 +9,19 @@ import Data.Monoid
 #endif
 import qualified Data.Text as T
 import Data.Typeable
+import Data.Semigroup
```
Comment (alevy) [inline src/Database/PostgreSQL/ORM/Validations.hs]: Just add a version gate for this so it works on older GHC versions as well.

File: src/Database/PostgreSQL/ORM/Validations.hs [hunk 2]
```diff
@@ -9,16 +9,19 @@ import Data.Monoid
...
 
 newtype ValidationError = ValidationError
   { validationErrors :: H.HashMap T.Text [T.Text] } deriving (Show, Typeable)
 
 instance Exception ValidationError
 
+instance Semigroup ValidationError where
```
Comment (alevy) [inline src/Database/PostgreSQL/ORM/Validations.hs]: Ditto for a version gate.

File: src/Database/PostgreSQL/ORM/Validations.hs (hunk 1 above)
Comment (ozzzzz) [reply src/Database/PostgreSQL/ORM/Validations.hs]: I hope now it is better

**Comment by ozzzzz** [comment]
If it is OK, could you merge it and upload to hackage (in perfect case - on Stackage too :) )? Thank you.

**Comment by ozzzzz** [comment]
Thank you :)

**Pull request merged**